frontend/spec/features/currency_spec.rb

```diff
@@ -14,7 +14,9 @@
     click_link "RoR Mug"
     click_button "Add To Cart"
     # Now that we have an order...
-    Spree::Config[:currency] = "AUD"
-    visit spree.root_path
+    with_unfrozen_spree_preference_store do
```

Comment:
I think we should document in the commit message why it's not possible to stub this preference (mainly because is not entirely clear to me yet :P ).

Reply:
I am also curious about this one.

Reply:
Being an old regression spec I wanted to keep the code as similar as the original as possible in order to avoid making the spec irrelevant by overlooking some details, so I did not stubbed it in the beginning, but after running this spec against the original code for spree/spree#2340 I think this can be safely stubbed as well. See also the fix for issue 2340: https://github.com/spree/spree/commit/58585fc55cad6027a4fa01a747e14999d5dc49ae

As far as I understand this spec basically ensured that this old line would find no order: 

```ruby
current_order = Spree::Order.find_by_id_and_currency(session[:order_id], current_currency, :include => :adjustments)
```

by changing the backend currency, thus making the line that followed (before the fix) raise an exception:

```ruby
current_order.last_ip_address = ip_address
```

I think this spec is not much relevant anymore, as that part of the code is long gone, so I guess it may be safely removed. What do you think? Maybe in another PR to keep things clean and focused here.